test/resolution-order-test.ts

```diff
@@ -86,13 +86,14 @@ test('Identifies `component:my-input` in a private collection from `template:/ap
       template: {
       },
       component: {
-        definitiveCollection: 'components'
+        definitiveCollection: 'components',
+        privateCollection: 'components'
```

Comment:
This seems like it should not be needed. A collection can always have a namespace (a name-prefix path) that results in a local resolution.

Rule 1 under: https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md#module-resolutions